msg/AttentionRecognitionTopNItem.msg

```diff
@@ -0,0 +1,2 @@
+float64 value
```

Comment:
@ekaszubski What is "value"?  Is this "likelihood"?  If so, I'd recommend changing to "likelihood" for clarity.  (Also, at some point, we will likely end up with "log_likelihood".)